IRP/src/CommonModules/Localization/Module.bsl

```diff
@@ -1073,6 +1077,13 @@ Function Strings(Lang) Export
 	Strings.Insert("Error_141", NStr("en = '[%1] cannot be changed, has posted documents'", Lang));	
 	Strings.Insert("Error_142", NStr("en = 'Wrong combination of send and receive debt type'", Lang));	
 	Strings.Insert("Error_143", NStr("en = 'Document Bank payment (currency exchange) not entered'", Lang));	
+	Strings.Insert("Error_144", NStr("en = 'Document''s movements were changed manually. Posting is denied'", Lang));
+	
+	// %1 - Register name
+	Strings.Insert("Error_145", NStr("en = 'There are differences between current movements and potential movements in register [%1]'", Lang));
```

Comment:
Differences detected between the current manual movements and the potential automated movements in register [%1].